<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmPurchasingGadget</class>
 <widget class="QDialog" name="ConfirmPurchasingGadget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirming purchase</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/check.png</normaloff>:/img/check.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{ 
   background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                                 stop:0.03 rgba(205,212,219,1), 
                                 stop:0.27 rgba(161,37,214,1), 
                                 stop:0.79 rgba(51,55,148,1));
}

QMessageBox{
	background-color: rgb(255, 255, 255);
	color:rgb(25, 25, 25);
	font: 350 12pt &quot;Bahnschrift SemiLight SemiConde&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_10">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_11">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:rgb(245, 245, 245);
	border:0px solid;
	border-radius:20px;
}

QComboBox{
	background-color:rgb(51, 55, 148);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(255, 255, 255);
	border:0px;
	border-radius:15px;
	padding-left:15px;
	
}

QComboBox:hover{
	background-color:rgb(28, 94, 156);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	padding-left:15px;
	border:0px solid rgb(0, 0, 0);
	border-radius:15px;
}

QLineEdit{
	background-color:rgb(51, 55, 148);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(255, 255, 255);
	border:0px;
	padding-left:15px;
	border-radius:15px;
}

QLineEdit:focus{
	background-color:rgb(67, 73, 194);
	color:rgb(255, 255, 255);
	border:2px solid rgb(0, 0, 0);
	border-radius:15px;
}



QPushButton{
border: solid 0px #ffffff;
background-color: rgb(36, 126, 210);
border-radius:18px;
color:rgb(255, 255, 255);
}

QPushButton:hover{
border: 1.5px inset rgb(1, 1, 1);
background-color: rgb(59, 104, 136);
border-radius:18px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Franklin Gothic Heavy</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:rgb(51, 55, 148);
border:0px;
margin-top:5px;
};</string>
        </property>
        <property name="text">
         <string>Detailed info:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="phoneNumberLn">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>38</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Phone number:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="gadgetBrandLn">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>38</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Gadget brand:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="gadgetModelLn">
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>38</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Gadget model:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="purchasePb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Franklin Gothic Heavy</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

QPushButton{
border: solid 0px #ffffff;
background-color: rgb(150, 39, 207);
border-radius:18px;
margin-left:63px;
margin-right:63px;
color:rgb(255, 255, 255);
}

QPushButton:hover{
border: 1.5px inset rgb(1, 1, 1);
background-color: rgb(109, 29, 152);
border-radius:18px;
}</string>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_9">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer_12">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
